docker/models/containers.py

```diff
@@ -709,7 +716,14 @@ def run(self, image, command=None, stdout=True, stderr=False,
         if exit_status != 0:
             stdout = False
             stderr = True
-        out = container.logs(stdout=stdout, stderr=stderr)
+
+        logging_driver = container.attrs['HostConfig']['LogConfig']['Type']
+
+        if logging_driver == 'json-file' or logging_driver == 'journald':
```

Comment:
Folks, I'm pretty sure there is always the better way doing something. The point of this PR was to overcome the pain of using not readable logging driver. But I'm wonder how we come to discussing why it was written this way in the past?

This is my first contribution in python project. Moreover this is the first time I'm programming python at all. I'm does not feel anouth courage to rewrite the whole method at this point.